endpoints/openrtb2/sample-requests/invalid-whole/array.json

```diff
@@ -1,4 +1,6 @@
 {
```

Comment:
`array.json`, `boolean.json`, `float.json`, `integer.json`, `null.json, and `empty-object.json` are all testing the same deserialization error. How about we keep just one (doesn't matter which) and rename it `malformed-request.json`?

Reply:
Done